Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md

```diff
@@ -0,0 +1,469 @@
+#  **Playing with Colors in CSS**
+
+Whenever we see a beautiful but simple website, we can feel that there is a definitely significant role of CSS behind it’s HTML files and who doesn’t know “colors” make everything attractive. The use of color is a fundamental form of human expression. Children experiment with color, before they even have the manual dexterity to draw. Maybe that's why color is one of the first things people often want to experiment with when learning to develop websites. Yes, so there must be some way to know about this and learn the basics, right? Who knows you might develop a keen interest about this! So, let’s get started.
+## 1. How to define colors in CSS?
+First, we have to know about this. In CSS, when we define properties for different HTML elements, we can give color values as their property in many cases too, like following:
+
+**p {** 
+
+ **color: red;**
+
+**}
+//for describing color for paragraph**
+
+There are four different ways to specify colors in CSS.
+
+•	“Color keywords” (eg. “Teal”, “Violet”, “Red”, “Yellow” etc.) ![](images/Screenshot%202021-09-28%20072549.jpg " Picture") 
+
+•	Using “RGB” functional notation (“RGB” stands for “Red, Green, Blue model” …. we can define any color in this model), eg. “color: rgb(255, 99, 71);” for “Tomato” color code.
+
+•	Using “HCL” functional notation (“HCL” stands for “Hue, Saturation, Luminosity model” …. we can define any color in this model), eg. “color: rgb(9, 100%,64%);” for “Tomato” color code.
```

Comment:
Use the proper unordered list in Markdown.
No need of unnecessary empty lines between list items.

Reply:
Keywords should be in inline code.